variables.tf

```
@@ -603,3 +603,9 @@ variable "existing_ingress_pcs_subnet_name" {
   type        = string
   default     = null
 }
+
+variable "ingress_service_name" {
+  description = "The name of the ingress service to attach the private link to."
+  type        = string
+  default     = "ingress-nginx-controller-internal"
```

i think this is actually supposed to just be `ingress-nginx-controller` now that we aren't using the `service.internal` values in the ingress-nginx helm chart.

```suggestion
  default     = "ingress-nginx-controller"
```